pkg/yurthub/certificate/hubself/cert_mgr.go

```diff
@@ -50,22 +50,21 @@ import (
 )
 
 const (
-	CertificateManagerName  = "hubself"
-	HubName                 = "yurthub"
-	HubRootDir              = "/var/lib/"
-	HubPkiDirName           = "pki"
-	HubCaFileName           = "ca.crt"
-	HubConfigFileName       = "%s.conf"
-	BootstrapConfigFileName = "bootstrap-hub.conf"
-	BootstrapUser           = "token-bootstrap-client"
-	DefaultClusterName      = "kubernetes"
-	ClusterInfoName         = "cluster-info"
-	KubeconfigName          = "kubeconfig"
+	yurtHubName             = "yurthub"
```

Comment:
how about keep the `yurtHubName` as `HubName`?  So it looks like variables names are consistent

Reply:
This variable's name has been changed back to hubname.